### PR TITLE
Update Maven config GAE 11

### PR DESCRIPTION
**Pull request by averikitsch**

### CODE REVIEW

File: appengine-java11/gaeinfo/README.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ Main class to your classpath:
 
 - Deploy to App Engine standard environment using the following Maven command.
 ```
-   mvn appengine:deploy -Dapp.deploy.projectId=<your-project-id>
+   mvn clean package appengine:deploy -Dapp.deploy.projectId=<your-project-id>
```
Comment (lesv) [inline appengine-java11/gaeinfo/README.md]: let's remove `-Dapp.deploy.projectId=<your-project-id>`

File: appengine-java11/kotlin-ktor/pom.xml [hunk 2]
```diff
@@ -15,173 +15,174 @@ See the License for the specific language governing permissions and
...
-                <scope>import</scope>
-            </dependency>
-        </dependencies>
-    </dependencyManagement>
+  <repositories>
+    <repository>
+      <id>repo1</id>
+      <url>https://jcenter.bintray.com</url>
+      <releases><enabled>true</enabled></releases>
+      <snapshots><enabled>true</enabled></snapshots>
```
Comment (lesv) [inline appengine-java11/kotlin-ktor/pom.xml]: Probably want that to be false -- we don't want snapshots

File: appengine-java11/kotlin-ktor/pom.xml [hunk 3]
```diff
@@ -15,173 +15,174 @@ See the License for the specific language governing permissions and
...
+    </repository>
+    <repository>
+      <id>repo2</id>
+      <url>https://kotlin.bintray.com/ktor</url>
+      <releases><enabled>true</enabled></releases>
+      <snapshots><enabled>true</enabled></snapshots>
```
Comment (lesv) [inline appengine-java11/kotlin-ktor/pom.xml]: we also don't want them here.

Comment (averikitsch) [reply appengine-java11/kotlin-ktor/pom.xml]: done

File: appengine-java11/gaeinfo/README.md (hunk 1 above)
Comment (averikitsch) [reply appengine-java11/gaeinfo/README.md]: done

File: appengine-java11/kotlin-ktor/pom.xml (hunk 2 above)
Comment (averikitsch) [reply appengine-java11/kotlin-ktor/pom.xml]: done

**Comment by averikitsch** [comment]
@lesv ready for review. Thanks

**Pull request merged**